modules/apps/roll-dialog/cast-dialog.js

```diff
@@ -69,6 +76,18 @@ export default class CastDialog extends SkillDialog {
         }
     }
 
+    async getData() 
+    {
+        let context = await super.getData();
+        if (this.userEntry.overchannelling) {
+            let overchannelling = parseInt(this.userEntry.overchannelling) ?? 0;
+            this.userEntry.overchannelling = overchannelling;
+            this.fields.overchannelling = overchannelling;
+            context.fields.overchannelling = overchannelling;
+        }
```

Comment:
Not sure what the point of this is? This should all be handled natively as any other field.

Reply:
There was a problem with Numeric fields for userEntry. Maybe it's now handled better in Library. The problem was, that overchannelling was always treated as string, and in result, onFieldChange was returning either NaN as text or undefined. I will test it out.

Reply:
resolved, it works fine without this change.